Controller/Annotations/AbstractScalarParam.php

```diff
@@ -69,9 +65,10 @@ public function getConstraints()
             if (false === $this->nullable) {
                 $constraints[] = new NotNull();
             }
-            if (false === $this->allowBlank) {
-                $constraints[] = new NotBlank();
-            }
+        }
+
+        if (false === $this->allowBlank) {
```

Comment:
I think my suggestion in #1769 was not good either. Keeping the constraint on line 59 is probably necessary so that it will be applied to the nested data. Though applying it to the collection itself does not make much sense IMO (and we also have the `NotNull` constraint).

Reply:
So should we just keep the `NotNull` additionnal check? I think it would make sense, I'm updating it pending your answer :)

Reply:
👍